tests/unit/StaffNotation.spec.ts

```diff
@@ -85,6 +86,7 @@ describe('StaffNotation', () => {
     expect(wrapper.find('.v-list').text()).not.toContain('Administrative Dissolution')
     expect(wrapper.find('.v-list').text()).not.toContain('Restore Company')
     expect(wrapper.find('.v-list').text()).toContain('Put Back On')
+    expect(wrapper.find('.v-list').text()).not.toContain('Administer Freeze')
```

Comment:
If business is historical (as in this test), should this say Administer Unfreeze instead? Please confirm with UX team.

Reply:
If the business is historical, Administer Freeze / Unfreeze will not appear as an option.

Reply:
OK, I understand.